src/components/Icon/iconRules.ts

```diff
@@ -35,7 +36,35 @@ const getIcon = (kind, name) => {
   return { content, fontFamily }
 }
 
-const getSize = size => `${sizes.get(size)}em` || '1em'
+const getSize = (size, isFontBased) =>
+  isFontBased ? `${sizes.get(size)}em` : `${sizes.get(size)}rem`
```

Comment:
Why ems for fonts and rems for non-fonts?

Reply:
this is a good observation - it seems that we could use the same principle of relative scaling for both. Will address this